index.html

```diff
@@ -52,14 +52,14 @@ <h2 class="project-tagline">QUIC support for Go</h2>
 
 <p>QUIC aims to solve the head-of-line blocking issue and the TLS connection round-trip problem. A detailed project description is available in the <a href="http://www.chromium.org/quic">QUIC project page</a>.</p>
 
-<p>Unfortunately, since QUIC is a experimental protocol, there's only one implementation of the protocol. And that implementation exists in project Chromium. Google provides toy server/client to play with. But the source code itself is not designed to be a library. It is heavily coupled to the existing Chromium sources.</p>
+<p>Unfortunately, since QUIC is an experimental protocol, there's only one client-side implementation of the protocol. And that implementation exists in project Chromium. Google provides toy server/client to play with. But the source code itself is not designed to be a library. It is heavily coupled to the existing Chromium sources.</p>
```

Comment:
Oh, I thought the `one` is about Chrome (since it is mentioned right after it), so I just wanted to clarify it as `client-side`.

Then, what about changing the words from:

```
Unfortunately, since QUIC is an experimental protocol, there's only one client-side implementation of the protocol. And that implementation exists in project Chromium. Google provides toy server/client to play with. But the source code itself is not designed to be a library. It is heavily coupled to the existing Chromium sources.
```

```
Unfortunately, there are few implementations available at this moment since QUIC is an experimental protocol. As far as we know, one server-side implementation and one client-side implementation exist, which is goquic and Chromium. Google provides toy server/client to play[...same as above]
```

If you are good with this, I'll proceed to make change and push it.